src/view/use-hidden-text-element/use-hidden-text-element.js

```diff
@@ -46,8 +46,11 @@ export default function useHiddenTextElement({
       getBodyElement().appendChild(el);
 
       return function unmount() {
-        // Remove from body
-        getBodyElement().removeChild(el);
+        // checking if element exists as the body might have been changed by thinks like 'turbolinks'
+        const body: HTMLBodyElement = getBodyElement();
```

Comment:
I found another culprit

Reply:
May want to change to things instead of thinks in the comment :)